src/api/apiClient.ts

```diff
@@ -35,5 +36,61 @@ axiosClient.interceptors.response.use(
     },
 )
 
+export const errorMap = (issue: ZodIssueOptionalMessage, ctx: ErrorMapCtx) => {
+    if (!ctx.data) {
+        return {
+            message: `${issue.path.join('.')} is a required field`,
+        }
+    }
+
+    switch (issue.code) {
+        case ZodIssueCode.too_small:
+            if (issue.type === 'string') {
+                return {
+                    message:
+                    `Invalid value: ${issue.path.join('.')} length must be >= ${issue.minimum} but got ${ctx.data}`,
+                }
+            } else {
+                return {
+                    message: 
+                    `Invalid value: ${issue.path.join('.')} must be >= ${issue.minimum} but got ${ctx.data}`,
+                }
+            }
```

Comment:
just a nitpick, but we could condense this by using a ternary operator
```suggestion
           return {
               message: issue.type === 'string'
                   ? `Invalid value: ${issue.path.join('.')} length must be >= ${issue.minimum} but got ${ctx.data}`
                   : `Invalid value: ${issue.path.join('.')} must be >= ${issue.minimum} but got ${ctx.data}`
            }
```

Reply:
good point - fixed!